mchf-eclipse/drivers/ui/menu/ui_menu_internal.h

```diff
@@ -92,6 +92,7 @@ enum
     MENU_METER_COLOUR_DOWN,
     MENU_DBM_DISPLAY,
     MENU_DBM_CALIBRATE,
+	MENU_FREQ_FONT,
```

Comment:
How to change  freq font? Im display menu I didnt see that.

Reply:
Which version of CPU do you have? This is available for 1024kB+ version.

Reply:
512kB :((

Reply:
Fonts needs memory. We had to limit unnecessary things to keep support for older machines, sorry.